series/type-int-list_test.go

```diff
@@ -0,0 +1,42 @@
+package series
+
+import "testing"
+
+func Test_intListElement_Set(t *testing.T) {
```

Comment:
Why is there no test for the operations supported by each type? e.g. `Greater`, `GreaterEq`  etc